subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -178,11 +178,10 @@ static uint8_t adv_sync_pdu_init_from(struct pdu_adv *pdu,
 		dptr_prev += sizeof(struct pdu_cte_info);
 	}
 
-	/* Copy AuxPtr, if applicable */
+	/* Add AuxPtr, if applicable. Do not copy since it will be updated later
```

Comment:
Please squash this commit with previous one. It is unnecessary to alter the same conde one commit after another.